README.md

```diff
@@ -59,18 +59,13 @@ solutions. If you cannot find your specific problem then please create a
 
 Adding a new component
 ---
-#### Install our [yeoman component generator](https://github.com/holidayextras/generator-ui-component)
```

Comment:
we should not be using this generator should we? It does not create anything like what we would recommend now does it?